client/methods/call.go

```diff
@@ -17,31 +17,33 @@
 package methods
 
 import (
-	log "github.com/eris-ltd/eris-logger"
-
 	"github.com/eris-ltd/eris-db/client"
 	"github.com/eris-ltd/eris-db/client/rpc"
 	"github.com/eris-ltd/eris-db/definitions"
 	"github.com/eris-ltd/eris-db/keys"
+	"github.com/eris-ltd/eris-db/util"
 )
 
 func Call(do *definitions.ClientDo) {
 	// construct two clients to call out to keys server and
 	// blockchain node.
-	erisKeyClient := keys.NewErisKeyClient(do.SignAddrFlag)
-	erisNodeClient := client.NewErisNodeClient(do.NodeAddrFlag)
+	logger, err := loggerFromClientDo(do, "Call")
+	if err != nil {
+		util.Fatalf("Could not generate logging config from ClientDo: %s", err)
```

Comment:
Yeah I agree I was working off the precedent with the exit thing.

As an aside, I don't like `Warn`s; they seem like they;ll be useful but they end up being a pain in the arse. I'd rather force the code author to do something drastic (error or exit) or just provide an informational message and make sure continued execution is valid, because I think providing a warning option gives implied license to do unsafe things because 'they were warned', when a lot of the time warnings are ignored/unobserved. Also various packages will 'warn' over things that can never be resolved (because of your architecture for example) so I think the stock of warnings are is pretty low.

In this case, we can return errors from the `methods` package, and `Fatalf` them in the `cmd` package. Don't see any problem with this, but then again when I see `ClientDo` I tend to think we should be able to assume we are at interactive context (and so are allowed to exit). So that is perhaps a separate argument for evicting ClientDo from this level (I'll see how that looks).